roles/kubernetes/preinstall/tasks/set_resolv_facts.yml

```diff
@@ -41,15 +41,27 @@
   set_fact: resolvconffile=/tmp/resolveconf_cloud_init_conf
   when: ansible_os_family == "CoreOS"
 
-- name: target dhclient conf/hook files for Red Hat family
+- name: check if /etc/dhclient.conf exists
+  stat: path=/etc/dhclient.conf
+  register: dhclient_stat
+
+- name: target dhclient conf file for /etc/dhclient.conf
   set_fact:
     dhclientconffile: /etc/dhclient.conf
+  when: dhclient_stat.stat.exists
```

Comment:
does it matter? If it's a symlink, it should point to the correct location and the task responsible for writing to it already has "follow: yes" set.